labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,43 @@
+var playerHealth = 40;
+var grantHealth = 10;
+var winCount = 0;
+
+function getDamage() {
+    damage = Math.floor(Math.random() * 5) + 1
+    return damage
+}
+
+function startCombat(damage) {
+   damage = getDamage();
```

Comment:
You're still struggling with arguments. 

An argument is a variable whose value gets set *when the function is called*:

```javascript
function double(aNumber) {
  return aNumber * 2;
}

double(6); // => 8
double(13); // => 26

function timesFive(aNumber) {
  const five = 5;
  return aNumber * five;
}

timesFive(3); // => 15
```

See how we never assign anything to `aNumber` *inside* `double`; we HAND numbers TO `double`, and whatever number we hand it, it assigns to `aNumber`.

We can declare new variables inside a function -- like in `timesFive` -- but they're not arguments, they're new! `five` isn't in the arguments list. (Note: assigning `5` to a variable named `five` is silly, but I did it to give you an example.)

So I would expect to see `damage` *either* in the arguments list, OR assigned in the first line, but definitely not both!